sdv/single_table/copulas.py

```diff
@@ -86,6 +88,7 @@ def __init__(self, metadata, enforce_min_max_values=True, enforce_rounding=True,
             field: self._validate_distribution(distribution)
             for field, distribution in (numerical_distributions or {}).items()
         }
+        self._num_rows = None
```

Comment:
why is this just always being set to None? Shouldn't we set this value when we fit?

Reply:
Yes I think I forgot to add it on the `fit`, I added it now.